Deprecate the use of high availability mode in STF (#494)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_high-availability.adoc

[id="high-availability_{context}"]
= High availability

[role="_abstract"]
With high availability, {Project} ({ProjectShort}) can rapidly recover from failures in its component services. Although {OpenShift} restarts a failed pod if nodes are available to schedule the workload, this recovery process might take more than one minute, during which time events and metrics are lost. A high availability configuration includes multiple copies of {ProjectShort} components, which reduces recovery time to approximately 2 seconds. To protect against failure of an {OpenShift} node, deploy {ProjectShort} to an {OpenShift} cluster with three or more nodes.

[WARNING]
====

> High availability (HA) mode is a deprecated feature. The use of HA mode is rarely required because {OpenShift} is a highly-available platform, and the enablement of HA mode can cause issues and make debugging issues more complicated. The use of HA mode is not supported in production environments. If you feel HA mode is required in your environment, a support exception is required.

Reviewer: ```suggestion
High availability (HA) mode is deprecated and is not supported in production environments. {OpenShift} is a highly-available platform and you can cause issues and complicate debugging if you enable HA mode. 
```

Author: Thanks for simplifying :) One suggestion to make it clear the issue lie in STF, not OCP:

High availability (HA) mode is deprecated and is not supported in production environments. {OpenShift} is a highly-available platform, and you can cause issues and complicate debugging in STF if you enable HA mode.

Reviewer: ProjectShort} high availability (HA) mode is deprecated and is not supported in production environments. {OpenShift} is a highly-available platform, and you can cause issues and complicate debugging in {ProjectShort} if you enable HA mode.
